bundles/net.tourbook.common/src/net/tourbook/common/color/Map3GradientColorManager.java

```diff
@@ -904,26 +905,17 @@ private static void readColors() {
 
       if (xmlColorDefinitions == null) {
 
-         for (final Map3ColorDefinition colorDef : _map3ColorDefinitions.values()) {
-
-            for (final Map3GradientColorProvider colorProvider : colorDef.getColorProviders()) {
-
-               final Map3ColorProfile colorProfile = colorProvider.getMap3ColorProfile();
-
-               colorProfile.setIsActiveColorProfile(true);
-
-               // set only first default color profile as active
-               break;
-            }
-         }
+         // set only first default color profile as active
+         _map3ColorDefinitions.values().forEach(colorDef -> colorDef.getColorProviders().get(0).getMap3ColorProfile().setIsActiveColorProfile(
+               true));
```

Comment:
I think the new code will fail with an NPE when `...get(0) == null`, this would not happen with the original code